introduction.md

```diff
@@ -2,3 +2,4 @@ Hello! I am here to teach people about GitHub.
 This could be iOS
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?